code/_onclick/cyborg.dm

```diff
@@ -17,7 +17,7 @@
 
 	if(is_ventcrawling(src)) // To stop drones interacting with anything while ventcrawling
 		return
-	if(stat == DEAD)
+	if(stat == DEAD || lockcharge || IsWeakened() || stunned || paralysis)
```

Comment:
```suggestion
	if(stat == DEAD || lockcharge || IsWeakened() || stunned || paralysis || low_power_mode)
```
Is it also worth adding low power mode to this too? They should only be able to do limited things in this mode.

Reply:
Please make this change too.